Mathlib/SetTheory/Ordinal/Topology.lean

```diff
@@ -227,4 +227,41 @@ theorem enumOrd_isNormal_iff_isClosed (hs : s.Unbounded (· < ·)) :
     rw [hb]
     exact le_bsup.{u, u} _ _ (ha.2 _ hba)
 
+open Set Filter
+
+/-- An ordinal is an accumulation point of a set of ordinals if it is positive and there
+are elements in the set arbitrarily close to the ordinal from below. -/
+def IsAcc (o : Ordinal) (S : Set Ordinal) : Prop :=
+  o ≠ 0 ∧ ∀ p < o, (S ∩ Ioo p o).Nonempty
```

Comment:
I may have asked this in the previous PR but, you do need the condition `o ≠ 0` for this match the topological notion, right? Getting rid of degenerate cases should only be done for good reasons.

Reply:
I added it originally so that `0` won't have to be in every club. It could also be a requirement in `IsClosed` instead, but yea then `IsAcc` won't match the topological notion. This felt like the more natural place to put it.

Reply:
Alright, glad to see this is a cut and dry case.